packages/fxa-auth-server/lib/senders/emails/templates/verifyPrimary/en.ftl

```diff
@@ -1,6 +1,6 @@
-verifyPrimary-title = Verify primary email
+verifyPrimary-title-2 = Confirm primary email
 verifyPrimary-description = A request to perform an account change has been made from the following device:
 verifyPrimary-subject = Confirm primary email
-verifyPrimary-action = Verify email
-verifyPrimary-action-plaintext = { verifyPrimary-action }:
-verifyPrimary-post-verify = Once verified, account changes like adding a secondary email will become possible from this device.
+verifyPrimary-action-2 = Confirm email
+verifyPrimary-action-plaintext = { verifyPrimary-action-2 }:
```

Comment:
```suggestion
verifyPrimary-action-plaintext-2 = { verifyPrimary-action-2 }:
```
This id will need to be updated, otherwise localized languages will call the out of date placeholder.